src/shogun/base/SGObject.h

```diff
@@ -353,6 +493,31 @@ class CSGObject
 	void unset_global_objects();
 	void init();
 
+	/** Checks if object has a parameter identified by a BaseTag.
+	 * This only checks for name and not type information.
+	 * See its usage in has() and has<T>().
```

Comment:
@karlnapf I tried testing it on local but I think docs are not generated for private methods. We may have to change the doxygen configuration file for this.